src/ducks/transactions/TransactionsPage.jsx

```diff
@@ -243,24 +242,20 @@ class TransactionsPage extends Component {
     }
 
     return (
-      <div
-        className={styles.TransactionPage__bottom + ' js-transactionPageBottom'}
```

Comment:
This element was used in https://github.com/cozy/cozy-banks/blob/master/src/ducks/transactions/Transactions.jsx#L136-L143, but I don't see the corresponding change in the PR. Is it normal? Does the infinite scroll still work?

Reply:
I change it with `js-scrolling-element`

Reply:
Oh yes sorry I missed the change in which you update the `querySelector`.